library/src/morpheus-master/sass/base/_extendables.scss

```diff
@@ -32,18 +31,17 @@
   border: 1px solid;
   border-left-width: 5px;
   border-color: $bordercolor;
-  --switch: calc(color.lightness(#{$color}) - var(--sakai-color-contrast-threshold)) * -100;
-  color: hsl(0, 0%, var(--switch));
```

Comment:
Unfortunately, I can't wrap my brain around how to make this work properly, so i removed it for now. I'm still on the hunt for ways to guarantee accessible color choices in our UI/theming.